### ISSUE TITLE
Chrome doesn't run on the heroku-16 stack

### ISSUE THREAD

**Issue by joshwlewis** [heroku/heroku-buildpack-google-chrome#12]
The common error looks like this:

```
/app/.apt/opt/google/chrome/chrome: error while loading shared libraries: libX11-xcb.so.1: cannot open shared object file: No such file or directory
```

For now, the workaround is to use the cedar-14 stack.

For Heroku CI, add this to your `app.json`:

```json
"stack": "cedar-14"
```

For a normal build:

```bash
heroku stack:set cedar-14
```

**Comment by joshwlewis** [comment]
Proposed fix is in #14.

**Comment by joshwlewis** [comment]
Closed by #14.
